### ISSUE TITLE
Double-click a GFA file to open it [macOS]

### ISSUE THREAD

**Issue by sjackman** [rrwick/Bandage#46]
I associated the GFA file type with Bandage.app. I double clicked a GFA file. Bandage opens, but the selected graph is not loaded. I suspect this is because the `Bandage.app/Contents/MacOS/Bandage` executable needs to be launched with the `Bandage load foo.gfa` command, whereas macOS is probably launching it with simply `Bandage foo.gfa` (unconfirmed). `open foo.gfa` similarly does not work. `Bandage load foo.gfa` works as expected.

**Comment by rrwick** [comment]
Yes, that would be a nice feature! It unfortunately seems to behaving differently on different platforms, so it might take a bit of work to make sure that implementing it for the Mac doesn't break Bandage on Linux/Windows!

For my own future reference: http://doc.qt.io/qt-5/qfileopenevent.html

**Comment by sjackman** [comment]
Thanks, Ryan!
